### PR TITLE
[minor] update SNI Client Support reference

### PR DESCRIPTION
**Pull request by antondollmaier**
noticed that the link to the SNIClientSupport-Page was broken.

### CODE REVIEW

File: source/configuration_manual/dovecot_ssl_configuration.rst [hunk 1]
```diff
@@ -125,7 +125,7 @@ If you have multiple IPs available, this method is guaranteed to work with all c
 With client TLS SNI (Server Name Indication) support
 ****************************************************
 
-It is important to note that having multiple SSL certificates per IP will not be compatible with all clients, especially mobile ones. It is a TLS SNI limitation. See SSL/SNIClientSupport for list of clients known to (not) support SNI.
+It is important to note that having multiple SSL certificates per IP will not be compatible with all clients, especially mobile ones. It is a TLS SNI limitation. See https://wiki.dovecot.org/SSL/SNIClientSupport for list of clients known to (not) support SNI.
```
Comment (cmouse) [inline source/configuration_manual/dovecot_ssl_configuration.rst:128]: it would be better to move the page too.

Comment (antondollmaier) [reply source/configuration_manual/dovecot_ssl_configuration.rst:128]: wasn't sure if that was desired - I'll transfer the contents. thx!

Comment (cmouse) [reply source/configuration_manual/dovecot_ssl_configuration.rst:128]: hi, did you have a chance to look at this?

Comment (antondollmaier) [reply source/configuration_manual/dovecot_ssl_configuration.rst]: thanks for your patience - copied contents now.

**Comment by sirainen** [comment]
Looks like this has been forgotten. Now it has conflicts, so I can't merge it. Can you rebase locally?

**Comment by antondollmaier** [comment]
> Can you rebase locally?

Done - made the changes on top of the latest commit.

**Review by sirainen — APPROVED** [review]

**Pull request merged**